CODE_OF_CONDUCT.md

```diff
@@ -6,7 +6,7 @@
 We as members, contributors, and leaders pledge to make participation in our
 community a harassment-free experience for everyone, regardless of age, body
 size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
+identity and expression, level of experience, education, socioeconomic status,
```

Comment:
We are based on the contributor covenant code of conduct. I would prefer to stick to the upstream text version.